assets/i18n/en.json

```diff
@@ -583,6 +585,8 @@
     "I18N_LEARNER_DASHBOARD_COMPLETED_TO_INCOMPLETE_COLLECTIONS": "<[numberMoved]> of the collections you completed have been moved to the 'in progress' section as new explorations have been added to them!",
     "I18N_LEARNER_DASHBOARD_CONTINUE_WHERE_YOU_LEFT_OFF_SECTION": "Continue where you left off",
     "I18N_LEARNER_DASHBOARD_CURRENT_GOALS_SECTION": "Current Goals",
+    "I18N_LEARNER_DASHBOARD_DECLINE_INVITATION_MODAL_BODY": "Are you sure you want to decline the joining request for <b><[groupName]></b> learner group?",
```

Comment:
Are you sure you want to decline to invitation to <learner group name>?

Reply:
done